tests/components/knx/test_diagnostic.py

```diff
@@ -29,70 +29,71 @@
 from tests.typing import ClientSessionGenerator
 
 
+@pytest.mark.parametrize("hass_config", [{}])
```

Comment:
This results in a slight change => hass_config will now be validated against the schema